### DISCUSSION TITLE
fPINN code

### DISCUSSION THREAD

**Discussion by ruirui-fu** [lululxvi/deepxde#1812] [Ideas]
How to understand what this code bcs_start means
<img width="379" alt="a295f9582644a121f48a6feda991511" src="https://github.com/user-attachments/assets/d657d0a3-2f75-4dc3-b458-b00d56743935">

**Comment by lululxvi** [comment]
See
https://github.com/lululxvi/deepxde/blob/d0c5aa25414ca469e07564d973e1ee499aff3c13/deepxde/data/pde.py#L55

**Reply by ruirui-fu** [reply]
Thank you!
Why is the number of points taken at the boundary 2 and 4 points taken when output? Take a look at the picture. And sometimes the output of the boundary number of points is not fixed, can the number of boundary points be fixed at the time of output?
<img width="508" alt="微信图片_20240920092224" src="https://github.com/user-attachments/assets/1cf7c354-3bd6-4ce1-bf77-4d2fcbdd5a21">

**Reply by lululxvi** [reply]
Boundary points are also used for PDE training.

**Reply by ruirui-fu** [reply]
Thank you very much!
